ex03.ex

```diff
@@ -101,8 +115,8 @@ defmodule Ex03 do
 
   """
 
-  def list_equal . . . "your code"
-
+  def list_equal([ h1 | t1 ], [ h1 | t1 ]), do: true
```

Comment:
# 0

This has a bug (it doesn't allow you to compare empty lists). But, as important, it doesn't actually use pattern matches. The first function head matches if `t1` == `t2`, so it is using the built-in list comparison.